sdks/python/container/py37/base_image_requirements.txt

```diff
@@ -17,7 +17,7 @@
 # Run ./gradlew :sdks:python:container:generatePythonRequirementsAll to update.
 # Do not edit manually, adjust ../base_image_requirements_manual.txt or
 # Apache Beam's setup.py instead, and regenerate the list.
-# You will need Python intepreters for all versions supported by Beam, see:
+# You will need Python interpreters for all versions supported by Beam, see:
```

Comment:
you'd have to fix this in the generator script

Reply:
Done.